Typecript - Cannot find module 'types/locale'

## Your Environment
* flatpickr version used: 4.5.1
* Typescript version used: 2.9.1

```ts
import flatpickr from 'flatpickr';
import { German } from "flatpickr/dist/l10n/de";

flatpickr('.fp', {
  locale: German,
});
```

Typescript error:
```
node_modules/flatpickr/dist/l10n/de.d.ts(57,14)
Cannot find module 'types/locale';
```

The error is correct because the path should be '../types/locale';

Thank you for this project 👍 

Best,
Simon

--- a/emitDeclarations.sh
+++ b/emitDeclarations.sh
@@ -8,3 +8,4 @@ else
 fi
 cp src/typings.d.ts dist/typings.d.ts
 rm -rf types
+find dist/types/l10n/ -type f -iname '*.d.ts' -exec sed -i '' -e 's#"types/locale"#"../types/locale"#g' "{}" +;

